proto/beacon/p2p/v1/types.proto

```diff
@@ -11,36 +10,34 @@ message BeaconState {
   repeated CrosslinkRecord crosslinks = 3;
```

Comment:
Can you please add sectional comments like this for `BeaconState`? It makes crosschecking the fields easier : )

https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#beaconstate